src/pages/settings/AboutPage/TroubleshootingPage.tsx

```diff
@@ -0,0 +1,105 @@
+import MenuItemList from '@components/MenuItemList';
+import TestToolMenu from '@components/TestToolMenu';
+import Text from '@components/Text';
+import TextLink from '@components/TextLink';
+import useEnvironment from '@hooks/useEnvironment';
+import useLocalize from '@hooks/useLocalize';
+import useTheme from '@hooks/useTheme';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import * as App from '@userActions/App';
+import * as Report from '@userActions/Report';
+import type {TranslationPaths} from '@src/languages/types';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {OnyxKey} from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+import SCREENS from '@src/SCREENS';
+
+const keysToPreserve: OnyxKey[] = [
+    ONYXKEYS.ACCOUNT,
+    ONYXKEYS.IS_CHECKING_PUBLIC_ROOM,
+    ONYXKEYS.IS_LOADING_APP,
+    ONYXKEYS.IS_SIDEBAR_LOADED,
+    ONYXKEYS.MODAL,
+    ONYXKEYS.NETWORK,
+    ONYXKEYS.SESSION,
+    ONYXKEYS.SHOULD_SHOW_COMPOSE_INPUT,
+    ONYXKEYS.NVP_TRY_FOCUS_MODE,
+];
+
+type BaseMenuItem = {
+    translationKey: TranslationPaths;
+    icon: React.FC<SvgProps>;
+    action: () => void;
+};
+
+function TroubleshootingPage() {
+    const {translate} = useLocalize();
+    const theme = useTheme();
+    const styles = useThemeStyles();
+    const {isProduction} = useEnvironment();
+
+    const wipeOnyxData = useCallback(() => {
+        Onyx.clear(keysToPreserve).then(() => {
+            App.openApp();
+        });
+    }, []);
+
+    const menuItems = useMemo(() => {
+        const baseMenuItems: BaseMenuItem[] = [
+            {
+                translationKey: 'initialSettingsPage.troubleshooting.resetAndRefresh',
+                icon: Expensicons.RotateLeft,
+                action: wipeOnyxData,
+            },
+        ];
+
+        return baseMenuItems.map((item) => ({
+            key: item.translationKey,
+            title: translate(item.translationKey),
+            icon: item.icon,
+            onPress: item.action,
+        }));
+    }, [translate, wipeOnyxData]);
+
+    return (
+        <IllustratedHeaderPageLayout
+            title={translate('initialSettingsPage.aboutPage.troubleshooting')}
+            onBackButtonPress={() => Navigation.goBack(ROUTES.SETTINGS_ABOUT)}
+            backgroundColor={theme.PAGE_THEMES[SCREENS.SETTINGS.TROUBLESHOOTING].backgroundColor}
+            illustration={LottieAnimations.Desk}
+        >
+            <View style={[styles.settingsPageBody, styles.ph5]}>
+                <Text style={[styles.textHeadline, styles.mb1]}>{translate('initialSettingsPage.aboutPage.troubleshooting')}</Text>
+                <Text style={styles.mb4}>
+                    <Text>{translate('initialSettingsPage.troubleshooting.description')}</Text>{' '}
+                    <TextLink
+                        style={styles.link}
+                        onPress={() => Report.navigateToConciergeChat()}
```

Comment:
would this work?
```
onPress={Report.navigateToConciergeChat}
```

Reply:
there is a type mismatch between those, so I'd leave it this way